consensus/core/src/mass/mod.rs

```diff
@@ -235,8 +235,9 @@ pub fn calc_storage_mass(
     //
     // Note: in theory this can be tighten by subtracting input mass in the process (possibly avoiding the overflow),
     // however the overflow case is so unpractical with current mass limits so we avoid the hassle
-    let harmonic_outs =
-        output_values.map(|out| storage_mass_parameter / out).try_fold(0u64, |total, current| total.checked_add(current))?; // C·|O|/H(O)
+    let harmonic_outs = output_values
+        .map(|(plurality, out)| storage_mass_parameter * plurality * plurality / out)
```

Comment:
I think it can overflow here, if SPK size is 430KB and then `10^9*(430000/100)^2 > 2^64`.

Reply:
Compute mass gives a weight of 11 gram to each spk byte, so max SPK size is ~45KB